libminifi/include/core/FlowConfiguration.h

```diff
@@ -134,6 +135,7 @@ class FlowConfiguration : public CoreComponent {
   utils::ChecksumCalculator& getChecksumCalculator() { return checksum_calculator_; }
 
  protected:
+  std::unordered_map<std::string, std::unique_ptr<ParameterContext>> parameter_contexts_;
```

Comment:
We could make this `gsl::not_null`, and not even store null contexts if they are even possible. And just give stronger guarantees to any code that uses this member.

Reply:
Updated in b7ea62e8d312dcb41c2893263cdbfcc46ab22064